projects/aca-shared/src/lib/components/tool-bar/toolbar-menu-item/toolbar-menu-item.component.ts

```diff
@@ -44,6 +45,9 @@ export class ToolbarMenuItemComponent {
   @Input()
   actionRef: ContentActionRef;
 
+  @ViewChild(MatMenuItem)
+  menuItem: MatMenuItem;
```

Comment:
Seems unused

Reply:
@AleksanderSklorz  It is used in `toolbar-menu.component.ts` here -> `menuItems.push(toolbarMenuItem.menuItem);`

Reply:
ah I see